packages/dev/parcel-namer-docs/package.json

```diff
@@ -4,9 +4,9 @@
   "private": true,
   "main": "DocsNamer.js",
   "engines": {
-    "parcel": ">= 2.0.0-alpha.3.2"
+    "parcel": "2.0.0-nightly.949"
   },
   "dependencies": {
-    "@parcel/plugin": "nightly"
+    "@parcel/plugin": "2.0.0-nightly.951"
```

Comment:
what's with the mismatch in nightlies?

Reply:
🤷 engines has to match the `@parcel/core` version. Not sure why `@parcel/plugin` is different. Lerna does weird things.